cli/src/torii/routing.rs

```diff
@@ -423,16 +425,38 @@ async fn handle_metrics(sumeragi: Arc<Sumeragi>, _network: Addr<IrohaNetwork>) -
 }
 
 #[cfg(feature = "telemetry")]
-#[allow(clippy::unused_async)] // TODO: remove?
-async fn handle_status(sumeragi: Arc<Sumeragi>, _network: Addr<IrohaNetwork>) -> Result<Json> {
-    // TODO: remove network
+#[allow(clippy::unnecessary_wraps)]
+fn handle_status(sumeragi: &Sumeragi) -> Result<warp::reply::Json, Infallible> {
     if let Err(error) = sumeragi.update_metrics() {
         iroha_logger::error!(%error, "Error while calling `sumeragi::update_metrics`.");
     }
     let status = Status::from(&sumeragi.metrics_mutex_access());
     Ok(reply::json(&status))
 }
 
+#[cfg(feature = "telemetry")]
+#[allow(clippy::unused_async)]
```

Comment:
Do we still need this handler to have `async`, even though the other ones got rid of it?

Reply:
Will fix in #3135.